honeybee_energy/properties/model.py

```diff
@@ -118,9 +125,9 @@ def check_duplicate_construction_set_names(self, raise_exception=True):
                 duplicate_names.append(con_set.name)
         if duplicate_names != []:
             if raise_exception:
-                raise ValueError('The model has the following duplicated '
-                                 'ConstructionSet names:\n{}'.format(
-                                    '\n'.join(duplicate_names)))
+                raise ValueError(
```

Comment:
I can't comment on the lines before this but as I commented on `honeybee-core` for all the duplicate checks like this you should use `set` for quick access.

Reply:
Thanks and good point. I will use `set`